benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -778,15 +834,36 @@ void GraphicsBenchmarkApp::RecordCommandBufferSpheres(PerFrame& frame)
 
 void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, size_t seed)
 {
-    if (pFullscreenQuadsColor->GetIndex() > 0) {
-        // Solid color
-        float3 colorValues = kFullscreenQuadsColorsValues[pFullscreenQuadsColor->GetIndex()];
-        frame.cmd->PushGraphicsConstants(mFullscreenQuads.pipelineInterface, 3, &colorValues);
-    }
-    else {
-        // Noise
-        uint32_t noiseQuadRandomSeed = (uint32_t)seed;
-        frame.cmd->PushGraphicsConstants(mFullscreenQuads.pipelineInterface, 1, &noiseQuadRandomSeed);
+    switch (pFullscreenQuadsType->GetIndex()) {
+        case 0: {
```

Comment:
nit: Consider making this an enum? There are a few places where an index is mentioned and we need to remember which index corresponds to which type

Reply:
OK, I added a enum class